src/models/User.ts

```diff
@@ -0,0 +1,31 @@
+import * as Sequelize from 'sequelize';
+import { sequelize } from './index';
+
+export interface UserAttributes {
+  name: string;
+}
+
+export interface UserInstance extends Sequelize.Instance<UserAttributes> {
+  name: string;
+
+  id: number;
+  createdAt: Date;
+  updatedAt: Date;
+}
+
+const User: Sequelize.Model<UserInstance, UserAttributes> = sequelize.define(
+  'User',
+  {
+    name: Sequelize.STRING,
+
+    // Google IDs are long af, so we need a longer type than INT
```

Comment:
this is just being pedantic, but in this case since the fact that Ids are strings, this should be part of the interface as well, and not just in the implementation

Reply:
Hmm I'm not really sure what you mean here. User instances do have an id property defined in `UserInstance`, so you could write something like:

```js
const user = await User.findById('random-id-string');
console.log(user.id) // -> 'random-id-string'
```

Does that still make it not part of the interface?